webpack/stories/docs/fetching-data-with-graphql.stories.mdx

```diff
@@ -0,0 +1,75 @@
+import { Meta } from '@theforeman/stories';
+
+<Meta
+  title="Introduction/Fetching data with GraphQL"
+  parameters={{
+    storyWeight: 160,
+  }}
+/>
+
+# Fetching data with GraphQL
+
+Foreman uses Apollo client to talk to GraphQL endpoint. All the components within react routes have access to the Apollo client as routes are wrapped with ApolloProvider (see ReactApp component for details). Queries can be executed simply by using a hook:
+
+```js
+import { useQuery, gql } from '@apollo/client';
+
+const myQuery = gql`
+  query {
+    whatever {
```

Comment:
Just a suggestion: wouldn't it be better to have an actual example? And add a comment that it should support whatever resource is currently supported by GraphQL, just to be over explicit...

Reply:
Thanks, I made an update as suggested.